### ISSUE TITLE
Vertical double bar not rendered in Firefox on Linux

### ISSUE THREAD

**Issue by prisae** [mathjax/MathJax#2741]
### Issue Summary

Vertical double bar (`\lVert`, `\rVert`) not rendered in Firefox on Linux.

### Steps to Reproduce:

See **Supporting information**.

### Technical details:

* MathJax Version: `src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"`
* Client OS: Ubuntu 20.04
* Browser: Firefox 90.0; Chrome 91.0

### Supporting information:

Example: https://emg3d.emsig.xyz/en/latest/api/emg3d.optimize.misfit.html

![Selection_001](https://user-images.githubusercontent.com/8020943/126490309-ce5be923-bb71-47b1-af66-eb8b18e28e40.png)

**Comment by dpvc** [comment]
I am not able to reproduce your result (on Mac OS; I need to set up an Ubuntu 20 VM).  Can you verify that you are getting version 3.2.0 and not an older cached version?  (Use the "About MathJax" item in the MathJax contextual menu.)

Also, does it appear if you zoom in or out?

**Comment by prisae** [comment]
Yes, it shows that it is version 3.2.0, both in Firefox and in Chrome. It works fine in Chrome/Chromium, but not in Firefox.

**Comment by dpvc** [comment]
Do the vertical bars appear if you change the zoom level?

**Comment by prisae** [comment]
Ha, yes!

**Comment by dpvc** [comment]
OK, thanks.  I'll have to install an Ubuntu VM to test further.

**Comment by prisae** [comment]
30/40/60/70/100 % do not work, the rest does.

**Comment by taniwallach** [comment]
I wonder if this is the twin of the problem with horizontal extenders. https://github.com/mathjax/MathJax/issues/2795

That was acknowledges to be a bug in Firefox  https://bugzilla.mozilla.org/show_bug.cgi?id=1741887  but it does not seem to be something which is high enough a priority for them to fix soon.

**Comment by prisae** [comment]
I just realized that the link I provided does not work any longer. Here an update:
https://emg3d.emsig.xyz/en/stable/api/emg3d.simulations.Simulation.html#emg3d.simulations.Simulation.misfit

However, the problem still exists.

**Comment by DestyNova** [comment]
I see similar behaviour in Firefox on Linux (but not Chromium) with single vertical bars in this formula:

```
$$ A = \frac{1}{2} \left\vert{\sum_{i=1}^n {(x_{i+1}+x_i)(y_{i+1}-y_i)}}\right\vert $$
```

They do reappear upon increasing the zoom to 110% or greater.

**Comment by dpvc** [comment]
The mechanism used to produce stretchy characters in v4.0 has been changed, and should produce more reliable results, sO I'm marking this as fixed.
